app/src/main/res/values/strings.xml

```diff
@@ -356,4 +354,33 @@
     
     <string name="preferred_player_fetcher_notification_title">Getting info…</string>
     <string name="preferred_player_fetcher_notification_message">"The requested content is loading"</string>
+
+    <!-- "What's New" feed -->
+    <string name="fragment_whats_new">What\'s New</string>
+    <string name="feed_first_time_loading">Loading news for the first time</string>
+    <string name="feed_last_updated">Last updated: <xliff:g id="update_time">%s</xliff:g></string>
+
+    <!-- Parsing YouTube upload dates in the format '2 days ago' -->
+    <string name="downloader_http_accept_language">en</string>
+    <string-array name="upload_date_seconds_ago_phrase">
```

Comment:
Why do you use string arrays?

Reply:
Some languages might not have a single, simple prefix that matches all forms of a word, like in English / Deutsch.

Reply:
Do you mean differences between singular and plurals form? Then I guess the `<plurals>` tag is the right choice here. Weblate (our translation service) will then take care of the differences in each language.

Reply:
Agree. The problems is there is no function `R.plurals.upload_date_phrases -> String[]`.
I would have to iterate over all possible integers that may change the plural form. Like this:
```java
for (int i = 0; i <= 20; i++)
    phrasesSet.add(recources.getQuantityString(R.plurals.upload_date_phrases, i));
```

The code is a little messy, but this will be certainly better for the translators because the plurals have an obvious meaning. I will implement it like this.